src/pages/ListEmpty/ListEmpty.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import './ListEmpty.css';
+
+const ListEmpty = () => {
+  let navigate = useNavigate();
+  const routeChange = () => {
+    let path = '/add-items';
+    navigate(path);
+  };
+  return (
+    <>
+      <h1>Smart Shopping List</h1>
+      <h2>Your shopping list is currently empty</h2>
+      <button onClick={routeChange}>Add Item</button>
```

Comment:
Aunque el resultado es el mismo, qué tal hacerlo con <Link> (sólo pensando que queda más corto y que de estar en vanilla JS se haría con <a href> en vez de una función, lo que sería equivalente).

```suggestion
      <button>
        <Link to="/add-items">Add Item</Link>
      </button>
```

Reply:
okis, si use este porque creo que ya en el proyecto habia una funcionalidad asi, y tal vez a futuro podria modularizarse, de todas maneras hice la observacion en este commit por si quieres evaluarlo 😊**113b82059f5cf4772c2f3bbae1ef44e76088d4e5**